mithril-aggregator/src/store/epoch_settings_storer.rs

```diff
@@ -7,33 +7,38 @@ use tokio::sync::RwLock;
 
 use mithril_common::entities::{Epoch, ProtocolParameters};
 
+use crate::entities::AggregatorEpochSettings;
+
 /// Store and get [protocol parameters][ProtocolParameters] for given epoch.
```

Comment:
update comment [protocol parameters][ProtocolParameters] to [epoch_settings][AggregatorEpochSettings]

Reply:
Done ✅